pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -38,6 +38,8 @@ def _strip_raw_data(tensor):
     meta_dict['type'] = "stripped"
     meta_dict['average'] = float(numpy.average(arr))
     meta_dict['variance'] = float(numpy.var(arr))
+    if not tensor.HasField("raw_data"):
+        tensor.raw_data = onnx.numpy_helper.from_array(arr, tensor.name).raw_data
     onnx.external_data_helper.set_external_data(tensor,
                                                 location=json.dumps(meta_dict),
                                                 length=len(tensor.raw_data))
```

Comment:
Instead of reconstructing `raw_data` how about reusing the `np.ndarray` field? :
```suggestion
                                                length=arr.nbytes)
```